### ISSUE TITLE
Can't use manipulator.

### ISSUE THREAD

**Issue by MaineBlender** [s-leger/archipack#184]
Archipack version: 1.2.83

Blender version: 2.8

Os version: 10.13

Description of the issue:
I'm working on a house project in Blender and for some reason, today, I can't adjust the sizes of some archipack elements using the manipulate dimension arrows or the hot corners of an object.  When I click on the yellow arrow, it reduces that dimension by a fixed amount and I can't hold the mouse button down to adjust it (and snap to a point to resize).  Nothing happens when I hold the mouse button down and try to adjust the size.  I also can't grab the hot corners and adjust or snap.  I did open a new blender file, created a wall and was able to slide the side in that file...  Grateful for any help.

**Comment by s-leger** [comment]
Double check that preferences window is closed, was an issue preventing manipulators from working on some blender 2.8 versions.

**Comment by MaineBlender** [comment]
Thanks, that window is definitely closed.

**Comment by s-leger** [comment]
You may try to select the object with failing manipulators, 
disable manipulate mode and re-enable.
Never encountered such issue before, so pretty hard to figure what's going wrong.

**Comment by MaineBlender** [comment]
Thanks, yeah I tried that too.  I tried it in a new blender document and it worked there...  It's pretty frustrating.

**Comment by s-leger** [comment]
Does object update when changing parameters ?  could be an "auto_update" issue after some errors.

**Comment by MaineBlender** [comment]
Not sure I know what you mean by changing parameters?  If I have a wall and I manually extend it by changing it's dimensions in the "item" menu, the dimensions shown by the manipulate feature do update.  When I added an extension to the wall (by adding a part), the angle arrow does work fine, but the arrows in the straight directions do not.  This is really an issue because when I tried to add a floor texture, I couldn't adjust the rectangle that was showing the pattern except by adjusting in the item menu, but then it also adjusted the scale, not just overall dimensions, so the patterning (for a hexagon floor) changed so that they weren't symmetrical hexagons anymore...

**Comment by MaineBlender** [comment]
Just tried it again in a new window and it's working fine.  I've just spent days on this 1 file and seems crazy to have to start over.

**Comment by s-leger** [comment]
In 1.2.x version you only are allowed to change size from one side, the yellow arrow, grey one will not work.
You may try to "append" all content in a new file, maybe this can solve some issues.

**Comment by MaineBlender** [comment]
Yes, I've only been trying from the yellow side, not the grey side.

**Comment by MaineBlender** [comment]
I think I'll make a duplicate file and slowly erase elements and see if I can isolate.

**Comment by MaineBlender** [comment]
s-leger-- So I had a window that was having trouble being rendered properly in the boolean feature.  I deleted the window and suddenly the manipulate features started working again...

**Comment by brianstever** [comment]
There's definitely an issue with the manipulator. A number of people are having the same issue, including me.

**Comment by s-leger** [comment]
Could you share a file where the error occurs ?
Also add more details about os / blender version so i'm able to repro.

**Comment by Althizar** [comment]
So I  had this issue for a good while, not sure if this is what corrected it but I switched from wireframe mode that I was currently in to a different mode and then they started to work again ill keep it posted if I notice anything else!

**Comment by s-leger** [comment]
Ensure that "gizmos" are enabled in 3d view, this setting may affect archipack's gizmos display too.
